content/developers/internals/data-model/usage-example.en.md

```diff
@@ -0,0 +1,278 @@
+
+The slope / curve value that one wants to set is constant on the whole range. For varying curves / slopes, one needs to create several objects.
+
+{{% alert title="" color="primary"%}}
+Mind that the `begin` value should always be smaller than the `end` value. That is why the curve / slope values can be negative: an uphill slope of 1 going from offset 10 to 0 is the same as a downhill slope of -1 going from offsets 0 to 10.
+{{% /alert %}}
+
+In the [small_infra.py](https://github.com/DGEXSolutions/osrd/blob/dev/core/examples/generated/scripts/small_infra.py) file, we have slopes on the track sections *TA6*, *TA7*, *TD0* and *TD1*.
+
+There are curves as well, on the track sections *TE0*, *TE1*, *TE3* and *TF1*.
+
+### Operational points
+
+`OperationalPoints` represent anything that we find of interest and want to represent in our simulation. An operational point is mostly constituted of the list of its parts, which are simply positions on track sections.
+
+For example, we might find convenient to have the position of the platforms actually integrated in our system. For that purpose, we can represent them as operational points.
+
+In our example infrastructure, they are the only operational points used. The operational point parts are represented by purple diamonds in the following schema, the operational points they are related to are simply the names they are close by.
+
+![Operational points examples](../svg_schemas/operational_points.drawio.en.svg)
+
+## Other objects
+
+Now we have represented all the material parts of our infrastructure <span style="color: red">really ?</span>. But there are still concepts that we may want to represent.
+
+### Loading Gauge Limits
+
+These objects are akin to `Slopes` and `Curves`: it covers a range of track section, with a `begin` and an `end` offset. It represents a restriction on the trains that can travel on the given range, by weight or by train type (freight or passenger).
+
+We did not put any in our examples.
+
+### Speed Sections
+
+The `SpeedSections` represent speed limitations (in meters per second) that are applied on some parts of the tracks. One `SpeedSection` can span on several track sections, and do not necessarily cover the whole track sections. Speed sections can overlap.
+
+In our example infrastructure, we have a speed section covering the whole infrastructure, limiting the speed to 300 km/h. On a smaller part of the infrastructure, we applied more restrictive speed sections. They are represented on the next image.
+
+![Speed section examples](../svg_schemas/speed_sections.en.svg)
+
+### Routes
+
+A `Route` in OSRD represents a trajectory a train take. It is represented with the following attributes:
```

Comment:
this is part of interlocking, and pretty important: a route is not a whole train path, only a fragment. It's an allowed move: without a route, trains can't go anywhere.
it should be moved to `Interlocking`